quesma/quesma/ui/console_routes.go

```diff
@@ -247,7 +252,9 @@ func (qmc *QuesmaManagementConsole) initPprof(router *mux.Router) {
 	router.HandleFunc("/debug/pprof/trace", pprof.Trace)
 }
 
-var store = sessions.NewCookieStore([]byte("test"))
+var authKey = securecookie.GenerateRandomKey(64)
```

Comment:
Do these values `64`, `32` ​​have a special meaning? I would suggest to add comment

Reply:
Added